includes/admin/class-dt-import.php

```diff
@@ -429,11 +432,13 @@ function getDefaultValues(id){
                                 jQuery('.value-mappers').each(function(){
                                     let h_sel = jQuery(this).data('value');
 
-                                    jQuery(this).find('option').each(function() {
-                                        if(h_sel === jQuery(this).text()){
-                                            jQuery(this).attr('selected','selected');
-                                        }
-                                    });
+                                    if (h_sel && h_sel.toLowerCase) {
```

Comment:
@cairocoder01 what does `h_sel.toLowerCase` here do?

Reply:
@corsacca When I was testing, I ran into a case where `h_sel` wasn't null/undefined, but it also wasn't a string. I can't remember how that actually happened. But the test to make sure `h_sel.toLowerCase` exists will essentially ensure that it is a valid string before trying to call `.toLowerCase()`.